### ISSUE TITLE
The subdivision of training and validation set

### ISSUE THREAD

**Issue by hellohelloriver** [RuiZhang97/ISNet#17]
In the paper, it says that the training, validation, and test set are set at a ratio of 5:3:2.
However, there is only a trainval.txt in the download link.
Could you please provide the subdivision of train and val?
